webapp/components/drawer/Drawer.js

```diff
@@ -123,27 +160,38 @@ export default ({
         widget={widget}
         disabled={widget.disabled}
         expanded={expand}
-        classes={classes}
+        classes={drawerClasses}
         createOrFocusWidget={createOrFocusWidget}
         status={status}
       />
     );
   };
-
+  const paperClasses = `${expand ? drawerClasses.openDrawer : drawerClasses.closeDrawer} ${classes.root}`;
   return (
-    <Paper elevation={0} className={expand ? classes.openDrawer : classes.closeDrawer}>
-      <div className={classes.container}>
+    <Paper elevation={0} className={paperClasses}>
+      <div className={drawerClasses.container}>
         <Box p={2}>
+          <Box className="drawerListBox">
+            <Typography variant="body2">Model Specification</Typography>
+          </Box>
           <List dense disablePadding>
-            {getMenu()
+            {getMenu()[0]
+              .map(mapItem)}
+          </List>
+          <Box className="drawerListBox">
```

Comment:
two different patterns in calling classes: classes.X and "X", would like to make this consistent, I think classes.X would be the way to go since it comes with static code checks and auto-completion, feel free to suggest otherwise, just aiming for consistency